webpack.config.js

```diff
@@ -211,7 +210,14 @@ module.exports = (env = {}) => {
         // set the current working directory for displaying module paths
         cwd: process.cwd(),
       }),
-    ],
+      // This must be added in dev mode if you want to do dev work
+      // on the service worker.
+    ].concat(
+      dev ? [] : [new InjectManifest({
+        swSrc: path.resolve(srcDir, 'serviceWorker/index.js'),
```

Comment:
With `path.resolve(srcDir, 'serviceWorker', 'index.js')` I believe it will add that included slash.